src/router.rs

```diff
@@ -68,27 +70,37 @@ pub mod tests {
             allowed_zome_fns.insert(allowed_zome_fn);
             let restricted_fns = AllowedFns::Restricted(allowed_zome_fns);
             allowed_fns.insert("coordinator".to_string(), restricted_fns);
-            let config =
-                Configuration::try_new("ws://127.0.0.1:1", "1024", "", allowed_fns, "", "")
-                    .unwrap();
-            Self::new_with_config(config)
+
+            let config = Configuration::try_new(
+                SocketAddr::new(Ipv4Addr::LOCALHOST.into(), 8888),
+                "1024",
+                "",
+                allowed_fns,
+                "",
+                "",
+            )
+            .unwrap();
+            Self::new_with_config(config).await
         }
 
-        pub fn new_with_config(config: Configuration) -> Self {
+        pub async fn new_with_config(config: Configuration) -> Self {
             let mut app_call = MockAppCall::new();
             app_call.expect_handle_zome_call().returning(|_, _| {
                 Box::pin(async {
                     let response = TestZomeResponse {
                         hello: "world".to_string(),
                     };
-                    Ok(ExternIO::encode(response)
-                        .map_err(|e| HcHttpGatewayError::RequestMalformed(e.to_string()))?)
+                    ExternIO::encode(response)
+                        .map_err(|e| HcHttpGatewayError::RequestMalformed(e.to_string()))
                 })
             });
 
+            // TODO This should be a mock. Done on another PR...
+            let admin_call = AdminConn::new(config.admin_socket_addr);
```

Comment:
For now, because app lookup and zome calls aren't implemented, this isn't getting used. So it doesn't do any harm. Will disappear on a rebase.